minecode_pipelines/miners/conan.py

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# purldb is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/aboutcode-org/purldb for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+from pathlib import Path
+import saneyaml
+
+from scanpipe.pipes.federatedcode import commit_changes
+from scanpipe.pipes.federatedcode import push_changes
+from minecode_pipelines import VERSION
+from minecode_pipelines.pipes.conan import store_conan_packages
+
+PACKAGE_BATCH_SIZE = 1000
+
+
+def mine_and_publish_conan_packageurls(conan_index_repo, cloned_data_repo, logger):
+    base_path = Path(conan_index_repo.working_dir)
+
+    yml_files = []
+    for file_path in base_path.glob("recipes/**/*"):
+        if not file_path.name == "config.yml":
+            continue
+        yml_files.append(file_path)
+
+    file_counter = 0
+    purl_files = []
+    purls = []
+
+    for idx, file_path in enumerate(yml_files, start=1):
+        package = file_path.parts[-2]
```

Comment:
Put a comment above that gives an example of what `file_path` looks like so we know what the -2 in the list slicing means